_posts/2022-06-28-trino-summit-call-for-speakers.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Announcing the 2022 Trino Summit"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+---
+
+We are pleased to announce the upcoming 2022 Trino Summit. The event is
+scheduled as *hybrid* event on the 10th of November 2022. You'll be able to join
+us online, or you can make the trip to San Francisco and meet us at the
+Commonwealth Club on the downtown waterfront.
+
+The event will be hosted by Starburst, one of the largest contributors to Trino.
+[Register now](https://www.starburst.io/info/trinosummit/) to attend (online
+*or* in-person).
+
+<!--more-->
+
+Perhaps more exciting, if you'd like to give a talk at this year's Trino summit,
+we're putting out a call for speakers. We will be accepting submissions from now
+until September 15th, but we recommend submitting soon, because slots are
+filling up fast. You can [submit your talk here](https://sessionize.com/trino-summit-2022/)
+once it's ready. We're looking for intermediate to advanced-level talks on a
+variety of themes, and you can check out the signup page for more information.
```

Comment:
again .. lets have all the necessary info here in this post as well..  we want to inspire people right here to submit .